honeybee_radiance/primitive/material/plastic.py

```diff
@@ -0,0 +1,259 @@
+"""Radiance Plastic Material.
+
+http://radsite.lbl.gov/radiance/refer/ray.html#Plastic
+"""
+from .materialbase import Material
+import honeybee_radiance.typing as typing
+
+
+class Plastic(Material):
+    """Radiance plastic material."""
+
+    def __init__(self, name, r_reflectance=0.0, g_reflectance=0.0, b_reflectance=0.0,
+                 specularity=0.0, roughness=0.0, modifier="void", dependencies=None):
+        """Create plastic material.
+
+        args:
+            name: Material name as a string. Do not use white space and special
+                character.
+            r_reflectance: Reflectance for red. The value should be between 0 and 1
+                (Default: 0).
+            g_reflectance: Reflectance for green. The value should be between 0 and 1
+                (Default: 0).
+            b_reflectance: Reflectance for blue. The value should be between 0 and 1
+                (Default: 0).
+            specularity: Fraction of specularity. Specularity fractions greater than 0.1
+                are not realistic (Default: 0).
+            roughness: Roughness is specified as the rms slope of surface facets. A
+                value of 0 corresponds to a perfectly smooth surface, and a value of 1
+                would be a very rough surface. Roughness values greater than 0.2 are not
+                very realistic. (Default: 0).
+            modifier: Material modifier (Default: "void").
+            dependencies: A list of primitives that this primitive depends on. This
+                argument is only useful for defining advanced primitives where the
+                primitive is defined based on other primitives. (Default: [])
+
+        Usage:
+            wall_material = Plastic("generic_wall", .55, .65, .75)
+            print(wall_material)
+        """
+        Material.__init__(self, name, modifier=modifier,
+                          dependencies=dependencies)
+        self.r_reflectance = r_reflectance
+        self.g_reflectance = g_reflectance
+        self.b_reflectance = b_reflectance
+        self.specularity = specularity
+        self.roughness = roughness
+
+        self._update_values()
+
+    def _update_values(self):
+        "update value dictionaries."
+        self._values[2] = [
+            self.r_reflectance, self.g_reflectance, self.b_reflectance,
+            self.specularity, self.roughness
+        ]
+
+        if self.specularity > 0.1:
+            print("Warning: Specularity values above .1 is uncommon for plastic.")
+        if self.roughness > 0.2:
+            print("Warning: Roughness values above .2 is uncommon.")
+
+    @property
+    def r_reflectance(self):
```

Comment:
That may end up being challenging as we have to come up with similar naming convention for all the other materials too. red / green / blue is the generic one that I took from Radiance documentation (https://floyd.lbl.gov/radiance/refer/ray.html) and  for glass and plastic that are the most used ones I made it more specific otherwise it is the generic red / green / blue naming.

Reply:
To me, the light and glow materials felt like the outliers since you already use reflectance and transmissivity everywhere else. I see it's especially helpful for glass and trans materials since `r_transmissivity` helps ensure people understand it is transmissivity and not transmittance.  I also got the sense that a lot of materials have separate values for reflection, transmission, emission that would require you to be specific about what type of RGB you are talking about (ie. you want to know if the RGB values you put in for a trans material are reflection and not transmission). And there are only 3 ways that light can interact with an object, right? (reflection, transmission, and emission). So I'm not sure that you will be really "coming up" with names here. You will just be explicit about what the input is.

Reply:
done.